core/save-profiledata.h

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: GPL-2.0
+#ifndef SAVE_PROFILE_DATA_H
+#define SAVE_PROFILE_DATA_H
+
+#include "dive.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+int save_profiledata(const char *filename, bool selected_only);
+void save_profiles_buffer(struct membuffer *b, bool select_only);
```

Comment:
Is there any plan to use this outside of save-profiledata.c? If not, then there seems to be no reason to export it.